bitcoin/src/util/psbt/error.rs

```diff
@@ -101,6 +103,7 @@ impl fmt::Display for Error {
             },
             Error::CombineInconsistentKeySources(ref s) => { write!(f, "combine conflict: {}", s) },
             Error::ConsensusEncoding => f.write_str("bitcoin consensus or BIP-174 encoding error"),
+            Error::NegativeFee => f.write_str("negative fee is not allowed"),
```

Comment:
Hmm I think more context like "the PSBT has a negative fee which is not allowed" would be easier to understand.

Reply:
ok, done